src/Spectre.Service.Tests/DataRootConfigTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using Spectre.Service.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spectre.Service.Tests
+{
+    [TestFixture]
+    class DataRootConfigTest
+    {
+        private string goodLocalPath  = @"\Path\To\Data";
+        private string wrongLocalPath = @"\Path\To\*";
+
+        private string goodRemotePath  = @"\\Path\To\Data";
+        private string wrongRemotePath = @"\\Path\To\*";
+
+        [Test]
+        public void DataRootThrowsOnIncorrectLocalPath()
```

Comment:
Either separate these tests (and define in the name of the test, what *Incorrect* means), or use `Assert.Multiple`.